src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Display from './Display';
+// import Button from './Button';
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.